### PR TITLE
Added some control-flow checks

### PR DESCRIPTION
**Pull request by harlanhaskins**

### CODE REVIEW

File: Tests/LLVMTests/IRBuilderSpec.swift [hunk 1]
```diff
@@ -197,9 +197,59 @@ class IRBuilderSpec : XCTestCase {
...
+      // CONTROLFLOW-NEXT: br i1 %1, label %then, label %else
+      builder.buildCondBr(condition: res, then: thenBB, else: elseBB)
+
+      // CONTROLFLOW: then:
+      builder.positionAtEnd(of: thenBB)
+
+      // CONTROLFLOW-NEXT: ret i32 1
+      builder.buildRet(IntType.int32.constant(1))
+
+      // CONTROLFLOW: else:
```
Comment (CodaFi) [inline Tests/LLVMTests/IRBuilderSpec.swift]: Does this and the then block name not follow immediately?

Comment (harlanhaskins) [reply Tests/LLVMTests/IRBuilderSpec.swift]: There's an empty line. Do you think I should add a
```
// CONTROLFLOW-NEXT:
```
before each of them?

Comment (harlanhaskins) [reply Tests/LLVMTests/IRBuilderSpec.swift]: It would be more complete that way.

**Comment by harlanhaskins** [comment]
@CodaFi Good to merge?

**Review by CodaFi — APPROVED** [review]

**Pull request merged**